src/pages/Vaults/Vault/RequestIssueModal/index.tsx

```diff
@@ -189,7 +189,8 @@ const RequestIssueModal = ({ onClose, open, collateralToken, vaultAddress }: Pro
       );
       // When requesting an issue, wait for the finalized event because we cannot revert BTC transactions.
       // For more details see: https://github.com/interlay/interbtc-api/pull/373#issuecomment-1058949000
-      const extrinsicResult = await submitExtrinsic(extrinsicData, FinalizedExtrinsicStatus);
+      const finalizedStatus = getExtrinsicStatus('Finalized');
```

Comment:
Would it be best make `Finalized` an `enum`? Also, I think it is a typo, instead it should be `Finalised`.

Reply:
This has to be 'Finalized' because that's how it is defined in the parachain.